language/use-cases/sql-code-generation/sql_code_generation.ipynb

```diff
@@ -0,0 +1,1609 @@
+{
```

Comment:
<p>Instead of commenting out the Colab only sections, use this to run it only if it detects colab as the environment</p><p> </p><pre class="ql-syntax" spellcheck="false">import sys

if "google.colab" in sys.modules:
    from google.colab import auth as google_auth

    google_auth.authenticate_user()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>